src/Components/Web/src/Forms/EditForm.cs

```diff
@@ -167,22 +167,27 @@ void RenderFormContents(RenderTreeBuilder builder, ModelBindingContext? bindingC
                 builder.AddAttribute(2, "action", bindingContext.BindingContextId);
             }
 
-            builder.AddMultipleAttributes(3, AdditionalAttributes);
-            builder.AddAttribute(4, "onsubmit", _handleSubmitDelegate);
+            if (bindingContext != null)
+            {
+                builder.AddAttribute(3, "method", "post");
```

Comment:
Is the intention that people should not be allowed to use other HTTP methods with an `EditForm`? AFAIK it would have worked with `GET` before - don't we want to allow that?

Reply:
This doesn't change that, does it?

It's only done when we are inside a form binding context, and we only support dispatching events when the browser sends a POST request already